package.json

```diff
@@ -32,20 +32,19 @@
     "grunt-prompt": "^1.3.3",
     "grunt-shell": "^2.1.0",
     "grunt-tslint": "^5.0.1",
-    "intern": "^3.4.3",
+    "intern": "~4.1.0",
     "load-grunt-tasks": "^3.5.2",
     "mockery": "^2.0.0",
     "sinon": "^2.1.0",
     "ts-node": "^3.0.2",
     "tslint": "^5.1.0",
     "tslint-sitepen": "^0.1.0",
-    "typescript": "^2.4.2"
+    "typescript": "~2.6.1"
   },
   "dependencies": {
-    "@dojo/core": "2.0.0-beta2.3",
-    "@dojo/has": "2.0.0-beta2.2",
-    "@dojo/interfaces": "2.0.0-beta2.3",
-    "@dojo/shim": "2.0.0-beta2.3",
+    "@dojo/core": "./tarballs/dojo-core-0.2.2-beta.1.tgz",
```

Comment:
We're not using @dojo for much in `src` we *could* update the code to remove these dependencies now or in a later PR.